generic/cells.cc

```diff
@@ -80,18 +80,16 @@ void lut_to_lc(const Context *ctx, CellInfo *lut, CellInfo *lc, bool no_dff)
     }
 
     if (no_dff) {
-        replace_port(lut, ctx->id("Q"), lc, ctx->id("Q"));
-        lc->params[ctx->id("FF_USED")] = 0;
+        replace_port(lut, ctx->id("Q"), lc, ctx->id("F"));
     }
 }
 
 void dff_to_lc(const Context *ctx, CellInfo *dff, CellInfo *lc, bool pass_thru_lut)
 {
-    lc->params[ctx->id("FF_USED")] = 1;
     replace_port(dff, ctx->id("CLK"), lc, ctx->id("CLK"));
 
     if (pass_thru_lut) {
-        lc->params[ctx->id("INIT")] = 2;
+        lc->params[ctx->id("INIT")] = 0xAAAA;
```

Comment:
This does make sense for architectures where the LUT inputs aren't default-0, but this needs to be generic across all LUT K (including the case of K>6 where an int64 can't even be used)

Reply:
Ouch, yes. Uhm... so should I revert it? Make it 64 bits? Make it some vector type?

Reply:
Have a look at how Property works - it already is a vector type, the int thing is just a convenient constructor